src/Core/backpropagation/include/BackupNegamax.h

```diff
@@ -0,0 +1,11 @@
+#include <BackupBase.h>
+
+class BackupNegmax : BackupBase {
+  public:
+    BackupNegmax();
+    using BackupBase::backup;
```

Comment:
Is this line necessary for anything?

Reply:
Sooo it is about that name hiding thing, as mentioned [here](https://stackoverflow.com/questions/16835897/overloading-base-class-method-in-derived-class). Problem was that I had to overload the backup function defined in backupBase, and without this it didn't want to compile.